serializer-base64/src/main/java/com/vlkan/hrrs/serializer/base64/Base64HttpRequestRecordWriter.java

```diff
@@ -5,45 +5,58 @@
 import java.io.ByteArrayOutputStream;
 import java.io.DataOutputStream;
 import java.io.IOException;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
 import java.util.List;
 
 import static com.google.common.base.Preconditions.checkNotNull;
 import static com.vlkan.hrrs.serializer.base64.Base64HttpRequestRecord.*;
 
 public class Base64HttpRequestRecordWriter implements HttpRequestRecordWriter<String> {
 
-    private final Base64Encoder encoder;
+    protected final Base64Encoder encoder;
+    protected final HttpRequestRecordWriterTarget<String> target;
+    private final DateFormat dateFormat = new SimpleDateFormat("yyyyMMdd-HHmmss.SSSZ");
 
-    private final HttpRequestRecordWriterTarget<String> target;
-
-    public Base64HttpRequestRecordWriter(HttpRequestRecordWriterTarget<String> target, Base64Encoder encoder) {
+    protected Base64HttpRequestRecordWriter(HttpRequestRecordWriterTarget<String> target, Base64Encoder encoder) {
         this.target = checkNotNull(target, "target");
         this.encoder = checkNotNull(encoder, "encoder");
     }
 
+    public static Base64HttpRequestRecordWriter createBase64HttpRequestRecordWriter(HttpRequestRecordWriterTarget<String> target, Base64Encoder encoder) {
+        if (System.getProperty("use.hrrs.queue") != null) {
+            return new QueueBase64HttpRequestRecordWriter(target, encoder);
```

Comment:
I cannot see who provides `QueueBase64HttpRequestRecordWriter`. IIRC, it is not included in `master`.